packages/sit-onyx/src/index.ts

```diff
@@ -1,5 +1,7 @@
 import "@/styles/index.scss";
 
+export { default as OnyxCheckbox } from "@/components/OnyxCheckbox/OnyxCheckbox.vue";
```

Comment:
We only ever wanted to use a single checkbox as a support component, so we shouldn't export it.

Reply:
Imho this is only true for the radio button. I can image several use cases where only one checkbox is needed. Examples:
- Custom content inside a table that can be checked
- "I agree to the terms of usage" with only 1 entry
- ...

Reply:
Right, but in that case you can also give a radio group with a single entry

Reply:
I would prefer to only have a single public component that has a name with checkbox.  if it doesn't work out we can still make it public again :)

Reply:
Alright I changed it :)